test-case/check-playback.sh

```diff
@@ -39,13 +39,17 @@ OPT_PARM_lst['s']=0             OPT_VALUE_lst['s']=1
 OPT_OPT_lst['F']='fmts'   OPT_DESC_lst['F']='Iterate all supported formats'
 OPT_PARM_lst['F']=0         OPT_VALUE_lst['F']=0
 
+OPT_OPT_lst['p']='rdtime'   OPT_DESC_lst['p']='sleep random duation between 0-7 seconds to cover runtime PM both active and suspend time'
```

Comment:
In fact, rdtime is a parameter to control random range, right?  if  rdtime is 10, the random range is 0~10, so the explanation here is not accurate.  change to --> `max random sleep time for runtime PM` is enough.